### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by zarva6596**
[DEMO LINK](https://zarva6596.github.io/react_static-list-of-posts/)

### CODE REVIEW

**Review by AlisaVlasova — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -24,6 +33,7 @@ const App = () => (
       <span>Users: </span>
       {users.length}
     </p>
+    <PostList postss={prepPosts} />
```
Comment (vkryvytskyy) [inline src/App.js:36]:
```suggestion
    <PostList posts={prepPosts} />
```

File: src/components/Comment/Comment.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Comment.scss';
+
+export const Comment = ({ name, email, body }) => (
+  <>
+    <div className="commentHeader">
```
Comment (vkryvytskyy) [inline src/components/Comment/Comment.jsx:7]: why don't you use BEM for classes?

File: src/components/CommentList/CommentList.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './CommentList.scss';
+import { Comment } from '../Comment';
+
+export const CommentList = ({ commentss }) => (
+  <ul className="commentsBox">
+    <h3 className="commentsHeader">Comments:</h3>
+    {commentss.map(comment => (
+      <li key={comment.id} className="commentsLi">
```
Comment (vkryvytskyy) [inline src/components/CommentList/CommentList.jsx:10]:
```suggestion
      <li key={comment.id} className="comments__comment comments__wrapper">
```

Comment (vkryvytskyy) [reply src/components/CommentList/CommentList.jsx:10]: or smth like that

File: src/components/Post/Post.jsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+      <div className="postBlock">
+        <div>
+          <h2 className="postName">{title}</h2>
+          <p className="postBody">{body}</p>
+        </div>
+        <User {...user} />
+      </div>
+    </div>
+
+    <CommentList commentss={comments} />
```
Comment (vkryvytskyy) [inline src/components/Post/Post.jsx:19]:
```suggestion
    <CommentList comments={comments} />
```
or those double 's' is your style?)

**Review by vkryvytskyy — APPROVED** [review]

**Pull request closed without merging**